core/server/data/schema/schema.js

```diff
@@ -576,6 +576,7 @@ module.exports = {
             defaultTo: 'pending',
             validations: {isIn: [['pending', 'submitting', 'submitted', 'failed']]}
         },
+        member_segment: {type: 'text', maxlength: 1000, nullable: true},
```

Comment:
Why was this specific length of `1000` chosen?

Reply:
I had the choice between small, medium and large string (https://github.com/TryGhost/Ghost/blob/ff37398311ea9c72fcad70efb78d8e90a6b37a38/core/server/data/schema/schema.js#L6) and I'm always cautious not to prevent use cases due to a too-restrictive database constraint.

Looking at a couple of examples I come up with, we could lower the limit the string to 191:
- `status:paid+email_count:>=3` (27 characters)
- `status:free+labels.slug:vip` (27 characters)

Reply:
do we have references to storing anything similar? I remember there was a problem at some point Kevin faced with the field being too small when storing a filter like this :thinking:

Reply:
After talking with Kevin, I've updated the issue with the reasoning behind using 2000 characters for this field.